pkg/server/event_handler_test.go

```diff
@@ -393,17 +394,32 @@ isOK: true`),
 			successMessenger: &testMessenger{
 				gotPmapEvent: &v1alpha1.PmapEvent{},
 			},
+			wantErrSubstr: "always fail",
 			wantPmapEvent: &v1alpha1.PmapEvent{},
 		},
 		{
-			name: "failed_process_and_send",
+			name: "failed_process_with_processErr",
+			notification: &pubsub.Message{
+				Attributes: map[string]string{"bucketId": "foo", "objectId": "pmap-test/gh-prefix/dir1/dir2/bar"},
+				Data:       testGCSMetadataBytes(),
+			},
+			gcsObjectBytes: []byte(`foo: bar
+isOK: true`),
+			processors: []Processor[*structpb.Struct]{&failProcessor{fmt.Errorf("processErr")}},
+			successMessenger: &testMessenger{
+				gotPmapEvent: &v1alpha1.PmapEvent{},
+			},
+			wantPmapEvent: &v1alpha1.PmapEvent{},
+		},
+		{
+			name: "failed_process_and_send_with_processErr",
 			notification: &pubsub.Message{
 				Attributes: map[string]string{"bucketId": "foo", "objectId": "pmap-test/gh-prefix/dir1/dir2/bar", "payloadFormat": "JSON_API_V1"},
 				Data:       testGCSMetadataBytes(),
 			},
 			gcsObjectBytes: []byte(`foo: bar
 isOK: true`),
-			processors: []Processor[*structpb.Struct]{&failProcessor{}},
+			processors: []Processor[*structpb.Struct]{&failProcessor{fmt.Errorf("processErr")}},
```

Comment:
Error attribute (the one we put in the pubsub attribute) is not checked in the test unless I'm missing something

Reply:
Now it's checked in test case `failed_process_and_send_with_processErr`